src/seq_mv/vector_batched.c

```diff
@@ -1185,8 +1185,8 @@ HYPRE_Int hypre_SeqVectorMassDotpTwo( hypre_Vector *x, hypre_Vector *y,
    {
       for (j = 0; j < k; j++)
       {
-         res_x = result_x[j];
-         res_y = result_y[j];
+	 res_x = 0; //result_x[j];
```

Comment:
Why are the comments needed here?  Which RHS is correct here?

Reply:
Good question. I am not sure. @PaulMullowney it seems that you made the changes.